src/WebJobs.Script/Content/Script/http/response.js

```diff
@@ -6,50 +6,73 @@ module.exports = (context) => {
         headers: {},
         body: undefined,
 
+        // functions methods
+        raw: (body) => {
+            res.isRaw = true;
+            return res.send(body);
+        },
+
+        // node httpResponse methods
+        setHeader: (field, val) => {
```

Comment:
`setHeader` and the like are methods that exist on node's httpResponse.  Express responses extend httpResponse with methods like `set` and `header`.  All methods are valid from an express point of view, got a customer who was confused at the lack of `setHeader` existence